governance/roles.md

```diff
@@ -76,6 +76,8 @@ navigate a complex security landscape. If the TAG has less than two Technical
 Leads, any Chair may act as Technical Lead.
 
 * Primary role of Chairs is to run operations and the governance of the group.
+  This includes coordination with the TOC and providing approval for governance
+  changes.
 * The Chairs are responsible for ensuring that group meetings are planned
```

Comment:
The responsibility of ensuring a meeting is facilitated effectively falls on the meeting facilitator based off the facilitation schedule. Sometimes chairs may be absent so that responsibility becomes shared with the designated person regardless of their role.